packages/fast-components/src/slider/slider.styles.ts

```diff
@@ -0,0 +1,135 @@
+import { css } from "@microsoft/fast-element";
+import { disabledCursor, display } from "../styles";
+import { focusVisible } from "../styles/focus";
+import { SystemColors } from "../styles/system-colors";
+
+export const SliderStyles = css`
+    :host {
+        --thumb-size: calc(var(--height-number) * 0.5);
+        --thumb-translate: calc(var(--thumb-size) * 0.5);
+        --track-overhang: calc((var(--design-unit) / 2) * -1);
+    }
+    :host([hidden]) {
+        display: none;
+    }
+
+    ${display("inline-grid")} :host {
+        align-items: center;
+        outline: none;
+        width: 100%;
+        margin: calc(var(--design-unit) * 1px) 0;
+        ${
+            /*
+           * Chromium likes to select label text or the default slot when
+           * the checkbox is clicked. Maybe there is a better solution here?
+           */ ""
+        } user-select: none;
+    }
+    .slider {
+        box-sizing: border-box;
+        border-radius: calc(var(--corner-radius) * 1px);
+        outline: none;
+        cursor: pointer;
+    }
+    .foreground-track {
+        display: none;
+        background: var(--neutral-foreground-hint);
+        transition: all 0.2s ease;
+    }
+    .layout-region {
+        position: relative;
+        margin: 0 8px;
+        display: grid;
+        grid-template-rows: calc(var(--thumb-size) * 1px) 1fr;
+    }
+    .thumb-container {
+        position: absolute;
+        height: calc(var(--thumb-size) * 1px);
+        width: calc(var(--thumb-size) * 1px);
+        transition: "all 0.2s ease";
+    }
+    .thumb-cursor {
+        border: none;
+        width: calc(var(--thumb-size) * 1px);
+        height: calc(var(--thumb-size) * 1px);
+        background: var(--neutral-foreground-rest);
+        border-radius: 50%;
+    }
+    .thumb-cursor:hover {
+        background: var(--neutral-foreground-hover);
+        border-color: var(--neutral-outline-hover);
+    }
+    .thumb-cursor:active {
+        background: var(--neutral-foreground-active);
+    }
+    .slider-horizontal .thumb-container {
+        transform: translateX(calc(var(--thumb-translate) * 1px));
+    }
+    .slider-vertical .thumb-container {
+        transform: translateY(calc(var(--thumb-translate) * 1px));
+    }
```

Comment:
Is this from a previous implementation or are we also supporting vertical with this first rev?

Reply:
this is from a previous iteration but, I was planning to have the styles and other things in place to support vertical orientation but create an issue for it not working and track that in this PR or separately.

Reply:
My only concern with this is that, without an implementation to map this too - we can't validate that it's a good solution for the problem. I'd prefer to implement it as part of the vertical orientation work because the code being here kind of implies that it's _good_ and _correct_.

Reply:
vertical orientation is in place now so this shall stay